### PR TITLE
fix(link): Delete link issue.

### PR DESCRIPTION
**Pull request by vikram-raj**
<strong>Note: If there are pending changes to the PR, prefix the PR title with "WIP" and add the label "DO NOT MERGE"</strong>

### Mandatory
 - [ ] What does this PR do? 
Fix delete link issue.
- [ ] What issue/task does this PR references?
https://github.com/openshiftio/openshift.io/issues/3601
- [ ] Are the tests Included?
Yes

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @vikram-raj for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin.

**Comment by vikram-raj** [comment]
[test]

**Comment by Raunak1203** [comment]
@vikram-raj Could you please add a test for this fix?

File: src/tests/specs/detailPage.spec.ts [hunk 1]
```diff
@@ -86,6 +86,19 @@ describe('Detail View test: ', () => {
...
+    let linkType = 'blocks',
+      searchWorkItem = '2-Workitem_Title_3',
+      Workitem_Title_3 = 'Workitem_Title_3';
+    await planner.workItemList.openDetailPage(c.workItemTitle2);
+    await planner.waitUntilUrlContains('detail');
+    await planner.detailPage.titleInput.untilTextIsPresentInValue(c.workItemTitle2);
+    await planner.detailPage.addLink(linkType, searchWorkItem, Workitem_Title_3);
+    expect(await planner.detailPage.getLinkedItems()).toContain(Workitem_Title_3);
+    await planner.detailPage.removeLink(Workitem_Title_3);
+    expect(await planner.detailPage.getLinkedItems()).not.toContain(Workitem_Title_3);
```
Comment (Raunak1203) [inline src/tests/specs/detailPage.spec.ts]: @vikram-raj test is expecting linkedListItem selector which is not there once you click close button 
expect(await planner.detailPage.getLinkedItems()).not.toContain(Workitem_Title_3);
you can replace it with links Count 
    await planner.detailPage.linkCount.untilTextIsPresent("0");
    expect(await planner.detailPage.linkCount.getTextWhenReady()).toBe("0");

and add selector in workitem-quickpreview.ts -
  linkCount = new ui.Clickable(this.$('#wi-link-total'), 'work item link total');

Comment (vikram-raj) [reply src/tests/specs/detailPage.spec.ts]: @Raunak1203 Resolved this issue.

**Pull request merged**